tests/functional/tests/bundles/smoke.yaml

```diff
@@ -0,0 +1,12 @@
+variables:
+  openstack-origin: &openstack-origin distro
+
+series: focal
+
+applications:
+  rabbitmq-server:
+    charm: ch:rabbitmq-server
+    num_units: 1
+    channel: 3.9/stable
```

Comment:
Should we maybe use `3.8/stable` channel so we can validate both same channel refreshing and channel switching?

Reply:
It would be nice to have, but since these will be done in func tests and do not think it's important. With this the charms will be changed and could not be put to previous state, so tests will failed in second run. Also these will take more time and smoke tests should be fast as possible + as Gabriel suggested we use smoke for running `cou run` and that will try to do same things.

I'm not saying it's not good, I'm opening just and discussion.